usr/src/uts/common/os/log_sysevent.c

```diff
@@ -641,6 +642,10 @@ sysevent_add_attr(sysevent_attr_list_t **ev_attr_list, char *name,
 		error = nvlist_add_hrtime(*ev_attr_list, name,
 		    se_value->value.sv_time);
 		break;
+	case SE_DATA_TYPE_NVLIST:
```

Comment:
@richardelling Can you clarify if this nvlist (as implemented) is "single-level" and thus compatible with syseventadm?  It doesn't have any nested nvlists.  But maybe you're saying that the whole concept of SE_DATA_TYPE_NVLIST is bad because existing consumers won't understand it?